apps/web/src/actions/evaluations/create.ts

```diff
@@ -11,7 +12,15 @@ export const createEvaluationAction = authProcedure
     z.object({
       name: z.string(),
       description: z.string(),
-      prompt: z.string().optional(),
+      type: z
+        .nativeEnum(EvaluationMetadataType)
+        .optional()
+        .default(EvaluationMetadataType.LlmAsJudge),
+      metadata: z
+        .object({
+          prompt: z.string(),
+        })
+        .optional(),
```

Comment:
from the FE pov evaluations always come with their metadata included, the FE should never have to deal with metadata tables